Src/BootCamp.Chapter/FileLogger.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.IO;
+
+
+namespace BootCamp.Chapter
+{
+    class FileLogger : Ilogger
+    {
+        public void Log(string message)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'Log' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5137406)